website/root.tsx

```diff
@@ -77,6 +77,7 @@ const mainClassname = css`
   box-sizing: border-box;
   height: 100vh;
   padding: 8px;
+  overflow: hidden;
```

Comment:
Without this the container height is not restricted and all the columns are rendered